appendix_meta_rules.md

```diff
@@ -490,7 +521,7 @@ action: include
 filename: other_sigma_rule.yml
 ```
 
-# Defeats
+# Filter
```

Comment:
```suggestion
# Suppression Filter
```
Just an idea for discussion. We already use the word filter for negative matches inside rules. To be able to address the filters by name, we could use "inline filter" and "suppression filter" to clarify if one means a specific one. Saying "filter" in general when the location doesn't matter in the discussion will also be fine but I think sometimes you want to be more specific.

If accepted, I will make changes in more locations, of course.